### PR TITLE
Test pdf reader

### PR DESCRIPTION
**Pull request by andhuynh**
- Implemented react-native-pdf to display a pdf reader to display zines pulled from the SRAA website
- Integrated slider using community library to control the scrolling of the PDF
- Added a button to jump to a certain page as a test button as a precursor to a future interactive table of contents
- Button also respectively changes the slider to respective position in the PDF
- Created zine cards with a button that leads to a details screen with the PDF reader
- Details screen has a back button going back to the card screen

### CODE REVIEW

File: mobile/App.js [hunk 1]
```diff
@@ -14,6 +16,7 @@ export default function App() {
         <Tab.Screen name="Map" component={MapScreen} />
         <Tab.Screen name="Gallery" component={GalleryScreen} />
         <Tab.Screen name="Submission" component={SubmissionScreen} />
+        <Tab.Screen name="Zines" component={ZinesScreen} />
       </Tab.Navigator>
     </NavigationContainer>
   );
```
Comment (he-is-james) [inline mobile/App.js:22]: For some reason, you don't have the stack navigator here? Might have been an error from a merge commit

File: mobile/Components/Zines.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
```
Comment (he-is-james) [inline mobile/Components/Zines.js:1]: I would rename this file to be `ZineCard` instead to be more specific

File: mobile/Screen/MapScreen.js [hunk 3]
```diff
@@ -1,37 +1,83 @@
...
+  //   console.log(URL);
+  //   try {
+  //     const result = await axios.post(`${URL}/test/post`, {
+  //       name: 'James', age: 20,
+  //     });
+  //     console.log(result.data);
+  //   } catch (err) {
+  //     console.error(err);
+  //   }
+  // };
```
Comment (he-is-james) [inline mobile/Screen/MapScreen.js:37]: I would remove this since this example will never be used again

File: mobile/Screen/MapScreen.js [hunk 4]
```diff
@@ -1,37 +1,83 @@
...
+    const value = Number(newValue);
+    setCurrentValue(value);
+    ref.setPage(value);
   };
 
   return (
     <View style={styles.container}>
       <Text>Map Screen</Text>
-      <Button title="Test" onPress={test} />
+      {/* <Button title="Test" onPress={test} /> */}
```
Comment (he-is-james) [inline mobile/Screen/MapScreen.js:51]: Remove comment

File: mobile/Screen/MapScreen.js [hunk 5]
```diff
@@ -1,37 +1,83 @@
-import React from 'react';
+import { React, useState } from 'react';
 import {
-  Button, StyleSheet, Text, View,
+  Dimensions, StyleSheet, Text, View,
 } from 'react-native';
-import axios from 'axios';
+// import axios from 'axios';
 // eslint-disable-next-line import/no-unresolved
```
Comment (he-is-james) [inline mobile/Screen/MapScreen.js:6]: Remove these comments

File: mobile/Screen/MapScreen.js [hunk 6]
```diff
@@ -1,37 +1,83 @@
...
-import { URL } from '@env';
+import Pdf from 'react-native-pdf';
+// import { URL } from '@env';
```
Comment (he-is-james) [inline mobile/Screen/MapScreen.js:8]: Remove comment

File: mobile/Screen/MapScreen.js [hunk 7]
```diff
@@ -1,37 +1,83 @@
...
     <View style={styles.container}>
       <Text>Map Screen</Text>
-      <Button title="Test" onPress={test} />
+      {/* <Button title="Test" onPress={test} /> */}
+      <Pdf
+        source={source}
+        ref={(pdf) => { ref = pdf; }}
+        horizontal
+        onLoadComplete={(numberOfPages) => {
+          console.log(`Number of pages: ${numberOfPages}`);
```
Comment (he-is-james) [inline mobile/Screen/MapScreen.js:57]: Remove `console.log` if unnecessary

File: mobile/Screen/ZineDetailsScreen.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {
+  StyleSheet, Text, View, ScrollView,
+} from 'react-native';
+import Zines from '../Components/Zines';
+
+function ZineDetailsScreen({ title, date }) {
+
```
Comment (he-is-james) [inline mobile/Screen/ZineDetailsScreen.js:8]: I think the `MapScreen` content should be moved here

File: mobile/Screen/ZinesScreen.js [hunk 9]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import {
+  StyleSheet, Text, View, ScrollView,
```
Comment (he-is-james) [inline mobile/Screen/ZinesScreen.js:3]: `Text` and `View` can be removed

**Review by he-is-james — CHANGES_REQUESTED** [review]
Everything looks really cool so far, excited to see what it will look when all put together!

Main issues are the extra folder `ios` and `MapScreen`/`ZineDetailsScreen`. Add `ios` to the `.gitignore` file and try to check the stack navigator from the `main` branch. Also remember to merge!

**Comment by andhuynh** [comment]
oops

**Pull request closed without merging**